src/constants.ts

```diff
@@ -1,7 +1,7 @@
 import { TMat2D } from './typedefs';
 
 export { version as VERSION } from '../package.json';
-export function noop() {}
+export function noop(...args: unknown[]): any {}
```

Comment:
necessary for types?

Reply:
Is typescript complaining that a function is either a function or default value noop?

Reply:
IDK but I did this so that I could use `noop` for functions that took an arbitrary number of params and returned anything. Specifically I needed it for the onChange type funcs in the animation.

Reply:
yes so is a typescript thing.
This thing of TS is bothering a lot to me, if you provided an hardcoded default value in the code, and it does not match with the other values...
I wonder why noop is not a recognized type of thing, since is an universal way to replace a function that does nothing.
Especially since onChange does not return anything